src/main/java/mods/railcraft/common/items/ItemCrowbar.java

```diff
@@ -52,7 +52,7 @@
 })
 public abstract class ItemCrowbar extends ItemTool implements IToolCrowbar, IBoxable, IToolWrench, IRailcraftItemSimple, IActivationBlockingItem {
 
-    private static final byte BOOST_DAMAGE = 3;
+    private static final int BOOST_DAMAGE = 3;
```

Comment:
For reference, you should know that I had some issue with neptunepink on the server several hours ago. My crowbar lost more than 1/3 durability just to trying to get the derailed carts back on to the track. I think it should go down.